tests/integration-service/README.md

```diff
@@ -0,0 +1,154 @@
+# Integration Service
+
+The Integration Service manages the lifecycle, deployment and integration of applications and components. It handles snapshots, environments, pipelines and deployments, ensuring applications run efficiently and are deployed correctly.
+
+
+## Happy Path Tests.
+
+### Basic Integration Tests:
+
+- Testing for successful creation of applications and components.
+- Asserting the successful creation of a snapshot after a push event.
+- Checking if the BuildPipelineRun is successfully triggered and completed.
+- Asserting the signing of BuildPipelineRun.
+- Validating the successful creation of a SnapshotEnvironmentBinding.
+
+
+### Pipeline and Integration Scenarios:
+
+- Verifying that all integration pipeline runs finish successfully.
+- Checking that a snapshot gets successfully marked as 'passed' after specific events.
+
+
+### Environment and Deployment Scenarios:
+
+- Ensuring that environments are created successfully.
+- Verifying the existence of deploymentTargetClass.
+
+
+## Negative Test Cases.
+
+### Failure Scenarios:
+
+- Creating an IntegrationTestScenario that is expected to fail.
+- Asserting that a snapshot is marked as failed.
+- Validating that no Release CRs and no SnapshotEnvironmentBinding are created in certain scenarios.
+- Checking that the global candidate does not get updated unexpectedly.
+
+
+### Non-existent Scenarios:
+- Verifying that entities like deploymentTargetClass and GitOpsCR don't exist under certain conditions.
```

Comment:
This test case is something similar like this -
### Failed ephemeral environment test case: 

When we are running a test case which will create an ephemeral environment, there is no deploymentTargetClass existing